cypress/integration/mongooseResources/category/edit/editRecord.spec.js

```diff
@@ -0,0 +1,61 @@
+import faker from 'faker';
+import { mongoose, leftNavbar } from '../../../../support/cssCommonSelectors';
+import { navbarTexts } from '../../../../support/texts';
+import comp from '../../../../support/components';
+
+const { inputs, boardView } = mongoose;
+
+describe('[Mongoose resources/ Category] Editing first record on the list', function () {
+  it('Should edit fields and check if changes are applied', function () {
+    let editedTitle = 'aaa' + faker.name.jobTitle();
```

Comment:
if the 'aaa' is needed to differentiate between records added by records it could be a helper function like `uniqe(faker.name.jobTitle())` and somewhere else:
```
const UNIQUE_CYPRESS_ID = 'aaa_im_from_cypress'
const unique = word => UNIQUE_CYPRESS_ID + word

Reply:
Deeper you go, there is less 'aaa'. It's just don't have more will to scream :smile: